registry/handlers/tags_list.go

```diff
@@ -0,0 +1,62 @@
+package handlers
```

Comment:
That is okay to have two separate dispatchers in the same file. Its nice to have everything related to tags in the same file.

I'd also have a not about what is going on here (redirect) + some historical context.